### PR TITLE
Small fixes and example production env file

### PR DESCRIPTION
**Pull request by sarayourfriend**

### CODE REVIEW

File: .env.example.prod [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+POSTGRES_DB=pra_request_tracker
+POSTGRES_USER=sjxHBcWufSbTJRVSCynqDavkxHefYxJP
+POSTGRES_PASSWORD=wgnuhcBthzKLm6e9NCUgMZXVlwjvEWNAzTqqA3LAmN7LcXzWmOWxjEgogCQghVMo
+
+AWS_STORAGE_BUCKET_NAME=prr-tracker
+AWS_ACCESS_KEY_ID=minio
+AWS_SECRET_ACCESS_KEY=minio123
+AWS_S3_REGION_NAME=us-west-2
+AWS_S3_ENDPOINT_URL=http://minio:9000
+MINIO_ROOT_PASSWORD=minio123
```
Comment (AetherUnbound) [inline .env.example.prod]: Should we maybe change these to AWS/DO Space values? And do we need the `MINIO_ROOT_PASSWORD` variable for prod?

File: .env.example.prod [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+DJANGO_DEFAULT_FROM_EMAIL=techblocsea@protonmail.com
+DJANGO_SERVER_EMAIL=error@openoversight.tech-bloc-sec.dev
+
+EMAIL_HOST=mail
+EMAIL_HOST_PASSWORD=terriblepassword
+EMAIL_HOST_USER=techblocsea@protonmail.com
+EMAIL_PORT=25
+EMAIL_USE_TLS=True
+EMAIL_SUBJECT_PREFIX="[Techbloc|OpenOversight]"
```
Comment (AetherUnbound) [inline .env.example.prod]:
```suggestion
EMAIL_SUBJECT_PREFIX="[Techbloc|PRA Request Tracker]"
```

File: .env.example.prod (hunk 1 above)
Comment (sarayourfriend) [reply .env.example.prod]: Oh you're right, I'll change that

File: .env.example.prod [hunk 3]
```diff
@@ -12,11 +12,10 @@ POSTGRES_USER=sjxHBcWufSbTJRVSCynqDavkxHefYxJP
...
 
 AWS_STORAGE_BUCKET_NAME=prr-tracker
-AWS_ACCESS_KEY_ID=minio
-AWS_SECRET_ACCESS_KEY=minio123
+AWS_ACCESS_KEY_ID=terriblkeyid
+AWS_SECRET_ACCESS_KEY=terriblekey
 AWS_S3_REGION_NAME=us-west-2
-AWS_S3_ENDPOINT_URL=http://minio:9000
-MINIO_ROOT_PASSWORD=minio123
+AWS_S3_ENDPOINT_URL=sfo3.digitaloceanspaces.com
```
Comment (AetherUnbound) [inline .env.example.prod]: It turns out (learned this the hard way :sweat_smile:) This has to start with the scheme
```suggestion
AWS_S3_ENDPOINT_URL=https://sfo3.digitaloceanspaces.com
```

**Review by AetherUnbound — APPROVED** [review]
Looks good!

**Pull request merged**